ocamldoc/odoc_misc.ml

```diff
@@ -507,7 +507,7 @@ let remove_option typ =
     | Types.Tvariant _
     | Types.Tpackage _ -> t
     | Types.Tlink t2
-    | Types.Tsubst t2 -> iter t2.Types.desc
+    | Types.Tsubst (t2, _) -> iter t2.Types.desc
```

Comment:
Can we assert `= None` here and in all other cases in `ocamldoc/`?

Reply:
It seems better to `assert false` on all `Tsubst` in all of ocamldoc.

Reply:
Done